linera-rpc/src/grpc/conversions.rs

```diff
@@ -548,6 +577,40 @@ impl From<api::Blob> for Blob {
     }
 }
 
+impl TryFrom<api::CertificateValue> for CertificateValue {
+    type Error = GrpcProtoConversionError;
+
+    fn try_from(certificate: api::CertificateValue) -> Result<Self, Self::Error> {
+        Ok(bincode::deserialize(certificate.bytes.as_slice())?)
+    }
+}
+
+impl TryFrom<CertificateValue> for api::CertificateValue {
+    type Error = GrpcProtoConversionError;
+
+    fn try_from(certificate: CertificateValue) -> Result<Self, Self::Error> {
+        Ok(Self {
+            bytes: bincode::serialize(&certificate)?,
+        })
+    }
+}
+
+impl TryFrom<HashedCertificateValue> for api::CertificateValue {
+    type Error = GrpcProtoConversionError;
+
+    fn try_from(hv: HashedCertificateValue) -> Result<Self, Self::Error> {
+        <HashedCertificateValue as Into<CertificateValue>>::into(hv).try_into()
```

Comment:
Maybe easier on the eyes:

```suggestion
        CertificateValue::from(hv).try_into()
```